scripts/assign.py

```
@@ -7,6 +7,123 @@
 import requests
 
 
+class PRCheckFailedError(RuntimeError):
+    """Raised when GitHub APIs could not be checked for existing PRs."""
+
+    pass
```

Ruff (0.15.2)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)